pkgs/by-name/ro/rosa/package.nix

```
@@ -2,16 +2,21 @@
 
 buildGoModule rec {
   pname = "rosa";
-  version = "1.2.39";
+  version = "1.2.43";
 
   src = fetchFromGitHub {
     owner = "openshift";
     repo = "rosa";
     rev = "v${version}";
-    hash = "sha256-K1FGiUNXSyCTmF//dculpnkTyn3hfqWrOiMUGk9kHrA=";
+    hash = "sha256-1/go7mL2uRzzV/iiTXsgZHGNW8EIONwEnb4jcMiKkv4=";
   };
   vendorHash = null;
 
+  patches = [
+    # https://github.com/openshift/rosa/pull/2326/
+    ./rosa-pr-2326.patch
```

Please don't include a file while the patch is available upstream.
```suggestion
    # Please remove at the next version bump.
    # See https://github.com/openshift/rosa/pull/2326/
    (fetchpatch {
      url = "https://github.com/openshift/rosa/commit/9ed236880f91f0e9514ba0a6f3be93ee115d1689.patch";
      hash = "";
    })
```

---

I have removed the patch file and used fetchpatch as per your suggestion. I've force pushed the changes into the previous commit

I was also waiting for this PR to pass before I let you know in the other PR. You were faster :)

Thanks a ton for your patience and your help!